packages/devtools_app/lib/src/flutter/screen.dart

```diff
@@ -76,8 +76,25 @@ abstract class Screen {
   }
 }
 
-/// Defines a page shown in the DevTools TabBar if [conditionalLibrary] is
+/// Defines a page shown in the DevTools [TabBar] if [conditionalLibrary] is
 /// present in the connected application.
+///
+/// If [conditionalLibrary] is not present, the screen will be hidden from the
+/// [TabBar], not just disabled like other DevTools [Screen]s.
+/// Example:
+///
+/// class PackageProviderScreen extends ConditionalScreen {
```

Comment:
Nit: see how the Flutter code includes examples in dartdoc comments to find how to make examples show up better in DartDoc.

Reply:
fixed